mapsims/tests/generate_cmb_case_data.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import print_function
+from pixell import enmap, powspec, curvedsky
+import numpy as np
+import os, sys
+from mapsims import cmb
+import healpy as hp
+
+try:
+    proceed = int(sys.argv[1])
```

Comment:
I renamed this file so it is not run automatically, I think we can remove this and just throw an error if run without arguments